gui/tray_applet/razer_tray_applet.py

```diff
@@ -430,7 +430,7 @@ def apply_startup_settings(self):
             if not start_brightness == 0:
                 self.daemon.set_brightness(start_brightness)
 
-            if start_macro:
+            if start_macro == 'true':
```

Comment:
Nope. the string is _currently_ either `'true'` or `'false'` as strings from JSON. So a mix of boolean would cause problems, particularly`if start_macro:` starts returning `True` if any string is present, as the previous code did.